### PR TITLE
Make dist_google support mutiple origins and destinations, SpatialPoints, return data.frame

### PR DESCRIPTION
**Pull request by Robinlovelace**
Heads up @richardellison I think this supecedes your PR - should have checked before doing this but I think this way is more generalisable. Feedback welcome but tested with new example using our `cents` data.

### CODE REVIEW

**Comment by richardellison** [comment]
Seems generally OK. Can you change the class(from) == "matrix" to "is(from, "matrix"), and the same for the other class inputs? Might also be worth adding support for dataframes as well. Simplest way would probably be to just add it to the matrix conditions.

**Comment by richardellison** [comment]
It also looks like you need to regenerate the documentation for the build to pass as everything looks OK otherwise.

**Comment by Robinlovelace** [comment]
Thanks for the feedback - the multimodal time estimates (potentially by time of day!) could be useful for modelling modal split for OD data - makes me think more about write-ups... Anyway I think I've fixed those issues and that this is ready to include. But any more changes needed, happy to fix.

File: R/google-functions.R [hunk 1]
```diff
@@ -46,18 +73,49 @@ dist_google <- function(from, to, google_api = "", g_units = 'metric'){
...
-  from = paste0(rev(from), collapse = ",")
-  to = paste0(rev(to), collapse = ",")
-  url <- paste0(base_url, g_units, "&origins=", from,
+  if(is(from, "matrix") | is(from, "data.frame"))
+    from = paste(from[,2], from[,1], sep = ",")
+  if(is(from, "numeric"))
+    from = paste(from[2], from[1], sep = ",")
+  if(is(to, "matrix") | is(to, "data.frame"))
+    to = paste(to[,2], to[,1], sep = ",")
+  if(is(from, "numeric"))
```
Comment (richardellison) [inline R/google-functions.R]: "from" should be changed to "to" on this line.

**Comment by richardellison** [comment]
Looks OK to me except for one line to fix (look for my inline comment). Other than that, looks good.

Travel times by time of day are certainly useful for modelling modal split.

**Comment by Robinlovelace** [comment]
Well spotted @richardellison - thanks for reviewing the code

**Comment by richardellison** [comment]
Looks good to me now, I'll merge and then remove my own PR.

**Pull request merged**